net.gwyddion.Gwyddion.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "net.gwyddion.Gwyddion",
+    "branch": "stable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gwyddion",
+    "rename-icon": "gwyddion",
+    "rename-desktop-file": "gwyddion.desktop",
+    "rename-appdata-file": "gwyddion.appdata.xml",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "modules": [     
+        "shared-modules/gtk2/gtk2.json",  
+        {
+            "name": "gwyddion",
+            "buildsystem": "autotools",
+	    "config-opts": [
+		
+	    ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://sourceforge.net/projects/gwyddion/files/gwyddion/2.53/gwyddion-2.53.tar.xz",
+                    "md5": "310733ff22ed978912195d5ac60b3bae"
```

Comment:
Any reasons for using a md5 hash?

Reply:
What alternative do I have?
https://sourceforge.net/projects/gwyddion/files/gwyddion/2.53/

Reply:
From project described hashes, none! You could manually run sha256sum or sha512sum on the file though if you wanted.